prov/rxm/src/rxm_ep.c

```diff
@@ -251,18 +247,15 @@ static int rxm_recv_queue_init(struct rxm_recv_queue *recv_queue, size_t size,
 	if (type == RXM_RECV_QUEUE_MSG) {
 		recv_queue->match_recv = rxm_match_recv_entry;
 		recv_queue->match_unexp = rxm_match_unexp_msg;
-		for (i = recv_queue->fs->size - 1; i >= 0; i--) {
-			recv_entry = &recv_queue->fs->buf[i];
-			recv_entry->comp_flags = FI_MSG | FI_RECV;
-		}
+		for (i = recv_queue->fs->size - 1; i >= 0; i--)
+			recv_queue->fs->buf[i].comp_flags = FI_MSG | FI_RECV;
 	} else {
 		recv_queue->match_recv = rxm_match_recv_entry_tagged;
 		recv_queue->match_unexp = rxm_match_unexp_msg_tagged;
-		for (i = recv_queue->fs->size - 1; i >= 0; i--) {
-			recv_entry = &recv_queue->fs->buf[i];
-			recv_entry->comp_flags = FI_TAGGED | FI_RECV;
-		}
+		for (i = recv_queue->fs->size - 1; i >= 0; i--)
+			recv_queue->fs->buf[i].comp_flags = FI_TAGGED | FI_RECV;
```

Comment:
Like the above change, can you please move the ones unrelated to multi recv to a separate commit?

Reply:
Did you forget to push the changes? I still see only one commit.